Update: Updates README.md with latest Docker image

tanubrata opened the pull request:
**Description:**

- Updates README.md with the link of latest stable JaBbA docker container and documentation on adding CPLEX to an existing container.

tanubrata committed:
Update: Updates Docker image w/ latest stable version in README

tanubrata committed:
Update: Updates Docker image w/ latest stable version in README

tanubrata committed:
Merge branch 'master' of https://github.com/tanubrata/JaBbA

Conflicts:
	README.md

tanubrata committed:
Fix: Updates the path for package copynumber

Conflicts:
	README.md

tanubrata committed:
Merge branch 'mskilab-org:master' into master

shihabdider reviewed (changes requested):
This is actually a good opportunity to fix another bug I found related to package installation. Basically, the `copynumber` package is no longer hosted on Bioconductor and needs to be installed from source the same way that `sequenza` is. Here is the url to the source file: https://www.bioconductor.org/packages/3.14/bioc/bin/macosx/contrib/4.1/copynumber_1.34.0.tgz

Please update the DESCRIPTION file to install from that url instead of from Bioconductor.

tanubrata committed:
Merge branch 'master' of https://github.com/tanubrata/JaBbA

tanubrata commented:
Ok, I have updated path for copynumber package with the source binary file. The one you pointed seems to be only for macosX, but I have added the binary. Looks good, let me know if there's anything else.
